src/palace/manager/core/opds2_import.py

```diff
@@ -741,6 +683,24 @@ def _extract_identifier(
         """
         return self.parse_identifier(publication.metadata.identifier)  # type: ignore[no-any-return]
 
+    @classmethod
+    def _extract_availability(
+        cls, availability: opds2_ast.OPDS2AvailabilityInformation | None
+    ) -> bool:
+        """Extract the publication's availability from its availability information.
+
+        :return: Boolean value indicating whether the publication is available.
+        """
+        available = opds2_ast.OPDS2AvailabilityType.AVAILABLE.value
+        if (
+            availability
+            and availability.state != available
```

Comment:
Because `since` implies past tense, I'm assuming that it is in the past. So since is just informational in the feed. Hadrien seemed strongly against `since` giving a state change in the future on the OPDS call.

Reply:
I can add a check that we fall within the range `since` -> `until` if you think that makes more sense. I was just basing the behaviour upon the discussion in the OPDS working group call.

Reply:
Fair enough. I'm not super happy with where we left that discussion and don't find compelling the arguments against representing future availability. That said, I agree with your other comment that we don't have enough information for other behavior.

Hadrien's comments there seem to imply that `availability.since` is, at best, informational (it shouldn't be actioned on for a future change); so, perhaps it _is_ best not to use it in any logic for now.

Reply:
I called out that `since` isn't used in my new comment, and I added a todo to revisit how we handle `since` once there is some resolution of your question here https://github.com/opds-community/drafts/discussions/63#discussioncomment-9806140